src/Uno.UI/UI/Xaml/UIElement.PointerCapture.cs

```diff
@@ -268,7 +292,7 @@ private void EnsureEffectiveCaptureState()
 
 					if (_nativeCaptureElement == null)
 					{
-						_nativeCaptureElement = _targets.Single().Key;
+						_nativeCaptureElement = _targets.Single().Value.NativeCaptureElement;
```

Comment:
Guaranteed to have a value ?

Reply:
Yes, otherwise I fallback to the `element` (so my fix doesn't apply)